nightfall-client/src/utils/computeCircuitInputs.mjs

```diff
@@ -115,6 +115,7 @@ export const computeCircuitInputs = (
     value,
     inputTokens,
     outputTokens,
+    sharedPubSender,
```

Comment:
Could you please use a variable name that is more intuitive?

Reply:
This is the shared secret for the sender (analog to public ephemeral key in previous version). The sender with this shared secret only has to apply a scalarMult with his private key to get the encryption key. So I put sharedPubxxxx in the variables that only need the private of xxxx to get the encryption key. I know it's a bit difficult to understand at the beginning.

Reply:
I think that `sharedPubKeySender` would be better. The way it is today only by diving into the code we can grasp about.